Doc/using/cmdline.rst

```diff
@@ -975,3 +975,12 @@ Debug-mode variables
    shutting down the interpreter.
 
    Need Python configured with the :option:`--with-trace-refs` build option.
+
+.. envvar:: PYTHONDUMPFILE
```

Comment:
"Dump file" is a generic name. What do you think of "Dump refs file" (PYTHONDUMPREFSFILE)? (and rename the PyConfig member to dump_refs_file).